src/main/java/com/thevoxelbox/voxelsniper/brush/type/rotation/Rotation2DBrush.java

```diff
@@ -20,9 +21,26 @@
 
     @Override
     public void handleCommand(String[] parameters, Snipe snipe) {
-        this.angle = Math.toRadians(Double.parseDouble(parameters[1]));
         SnipeMessenger messenger = snipe.createMessenger();
-        messenger.sendMessage(ChatColor.GREEN + "Angle set to " + this.angle);
+        String firstParameter = parameters[0];
+
+        if (firstParameter.equalsIgnoreCase("info")) {
+            messenger.sendMessage(ChatColor.GOLD + "Rotation2D Brush Parameters:");
+            messenger.sendMessage(ChatColor.AQUA + "/b rot2 [a] -- Sets rotation angle to a.");
```

Comment:
That should be n
How does it parse inputs?
![](https://i.imgur.com/BXkemFT.png)

Reply:
Angle is converted to gradians, I'll add a note after the value.

Reply:
Putting this here, if you find some other args that aren't `n` but represent a number (therefore aren't a needed parameter), please replace it for sake of consistency. Having too many aliases for representations in place is just confusing.